### ISSUE TITLE
4장: 분류

### ISSUE THREAD

**Issue by hyunmin0317** [hyunmin0317/ML-Study#4]
### 4.2. 결정 트리
* 결정 트리 모델의 특징
* 결정 트리 파라미터
* 결정 트리 모델의 시각화
* 결정 트리 과적합(Overfitting)
* 결정 트리 실습 – 사용자 행동 인식 데이터 세트
### 4.3. 앙상블 학습
* 앙상블 학습 개요
* 보팅 유형 – 하드 보팅(Hard Voting)과 소프트 보팅(Soft Voting)
* 보팅 분류기(Voting Classifier)
### 4.4. 랜덤 포레스트
* 랜덤 포레스트의 개요 및 실습
* 랜덤 포레스트 하이퍼 파라미터 및 튜닝
### 4.5. GBM(Gradient Boosting Machine)
* GBM의 개요 및 실습
* GBM 하이퍼 파라미터 및 튜닝
### 4.6. XGBoost(eXtra Gradient Boost)
* XGBoost 개요
* XGBoost 설치하기
* 파이썬 래퍼 XGBoost 하이퍼 파라미터
* 파이썬 래퍼 XGBoost 적용 – 위스콘신 유방암 예측
* 사이킷런 래퍼 XGBoost의 개요 및 적용
### 4.7. LightGBM
* LightGBM 설치
* LightGBM 하이퍼 파라미터
* 하이퍼 파라미터 튜닝 방안
* 파이썬 래퍼 LightGBM과 사이킷런 래퍼 XGBoost, LightGBM 하이퍼 파라미터 비교
* LightGBM 적용 – 위스콘신 유방암 예측
### 4.8. 분류 실습 – 캐글 산탄데르 고객 만족 예측
* 데이터 전처리
* XGBoost 모델 학습과 하이퍼 파라미터 튜닝
* LightGBM 모델 학습과 하이퍼 파라미터 튜닝
### 4.9. 분류 실습 – 캐글 신용카드 사기 검출
* 언더 샘플링과 오버 샘플링의 이해
* 데이터 일차 가공 및 모델 학습/예측/평가
* 데이터 분포도 변환 후 모델 학습/예측/평가
* 이상치 데이터 제거 후 모델 학습/예측/평가
* SMOTE 오버 샘플링 적용 후 모델 학습/예측/평가
### 4.10. 스태킹 앙상블
* 기본 스태킹 모델
* CV 세트 기반의 스태킹
### 4.11. 정리
